src/lib/validation/limit-cart-items.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+import constants from '../constants';
+
+export const limitCartItems = (input : any)  => {
+  const { cartLimit } = constants;
+  if (input.items.length > cartLimit) {
```

Comment:
what if `items` is undefined? I would do this:
```
const items = input.items || []
if (items.length > cartLimit) {
  logger.....
}
input.items = items
```
Also.... we are using a `logger`, look at the other files to see how its used.

Reply:
yep just added a check for (input.items && items.length > cartLimit)